src/gam/__init__.py

```diff
@@ -11203,7 +11203,7 @@ def ProcessGAMCommand(args):
             elif argument in ['gcpfolder']:
                 createGCPFolder()
             elif argument in ['adminrole']:
-                gapi_directory_roles.create()
+                gapi_directory_roles.createUpdate(False)
```

Comment:
Let's keep separate create and update functions even if they are simple wrappers to the main function.